### PR TITLE
Update Documentation for recent Figma and DesignCompose changes

### PR DESCRIPTION
**Pull request by timothyfroehlich**
Also fix the syntax highlighting for shell commands

### CODE REVIEW

**Comment by timothyfroehlich** [comment]
Whoops, MediaCompose changed a bit more than I thought. Need some more updates there.

**Comment by timothyfroehlich** [comment]
All good now.

File: docs/_docs/media-center-demo/index.md [hunk 1]
```diff
@@ -181,7 +159,7 @@ Therefore, all variants must be signed.
     the key file.
 
 1.  In
-    `reference-apps/aaos-unbundled/standalone-projects/mediacompose/app/build.gradle.kts`,
+    `reference-apps/aaos-unbundled/mediacompose/app/build.gradle.kts`,
```
Comment (rylin8) [inline docs/_docs/media-center-demo/index.md:162]: There's no 'app' folder anymore: https://github.com/google/automotive-design-compose/tree/main/reference-apps/aaos-unbundled/mediacompose

File: docs/_docs/media-center-demo/index.md [hunk 2]
```diff
@@ -220,10 +198,10 @@ app][17].
 Update the Media Center app to load your copy of the Figma Design:
 
 1.  Open the app's `MainActivity.kt` file, located within the Android project at
-    `MediaCompose/mediacompose/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
+    `DC Unbundled Apps/mediacompose/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
```
Comment (rylin8) [inline docs/_docs/media-center-demo/index.md:201]: Think this should be `reference-apps/aaos-unbundled/mediacompose/src/main/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`

File: docs/_docs/media-center-demo/index.md [hunk 3]
```diff
@@ -220,10 +198,10 @@ app][17].
...
 
     The actual file location is
-    `reference-apps/aaos-unbundled/standalone-projects/mediacompose/app/src/main/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
+    `reference-apps/aaos-unbundled/mediacompose/app/src/main/java/com/android/designcompose/reference/mediacompose/MainActivity.kt`.
```
Comment (rylin8) [inline docs/_docs/media-center-demo/index.md:204]: no `app` here

File: docs/_docs/media-center-demo/index.md [hunk 4]
```diff
@@ -292,7 +270,7 @@ to the app's source for inclusion in the APK.
     render.
 
 1.  Create a directory in
-    `reference-apps/aaos-unbundled/standalone-projects/mediacompose/app/src/main`
+    `reference-apps/aaos-unbundled/mediacompose/app/src/main`
```
Comment (rylin8) [inline docs/_docs/media-center-demo/index.md:273]: no `app`

File: docs/_docs/media-center-demo/index.md [hunk 5]
```diff
@@ -321,7 +299,7 @@ to the app's source for inclusion in the APK.
 
 1.  Download the cached documents and copy them into the new assets directory
     created earlier
-    (`reference-apps/aaos-unbundled/standalone-projects/mediacompose/app/src/main/assets/figma`).
+    (`reference-apps/aaos-unbundled/mediacompose/app/src/main/assets/figma`).
```
Comment (rylin8) [inline docs/_docs/media-center-demo/index.md:302]: no `app`

File: docs/_docs/media-center-demo/index.md (hunk 3 above)
Comment (timothyfroehlich) [reply docs/_docs/media-center-demo/index.md:204]: Gah. Fixed.

File: docs/_docs/media-center-demo/index.md (hunk 2 above)
Comment (timothyfroehlich) [reply docs/_docs/media-center-demo/index.md:201]: This line is referring to the location of the file within the Android view in Android Studio. The fild location is mentioned on the next line.

**Review by rylin8 — APPROVED** [review]

**Pull request merged**